tests/integration.rs

```diff
@@ -4017,28 +4016,47 @@ mod tests {
                     .does_device_vendor_pair_match("0x1057", "0x1af4")
                     .unwrap_or_default());
 
-                // Verify the first disk is located under IOMMU group 0.
+                // For AArch64, now virtual IOMMU is only tested on FDT, not ACPI.
+                // In the case of FDT, the behavior of IOMMU is a bit different with ACPI. All the
+                // devices on the PCI bus will be attached to the virtual IOMMU, except the
+                // virtio-iommu device itself. So these devices will all be added to IOMMU groups,
+                // and appear under folder '/sys/kernel/iommu_groups/'.
+                // The result is, on AArch64 IOMMU group '0' contains "0000:00:01.0" which is the
+                // console. The first disk "0000:00:02.0" is in group '1'.
+                // While on X86, console device is not attached to IOMMU. So the IOMMU group '0'
+                // contains "0000:00:02.0" which is the first disk.
```

Comment:
I understand the difference of behavior for FDT, which sounds fine to me, but I'd like to see a note in the documentation `docs/iommu.fd` about it. Basically, for FDT, as soon as you enable the vIOMMU with one device, all the other devices are attached to it. This is not very intuitive wrt the CLI.
/cc @rbradford

Reply:
Good idea. This special behavior should be mentioned in doc.

Reply:
@sboeuf , I added a commit to update the document, PTAL.